components/track/TrackItem.vue

```diff
@@ -69,6 +67,7 @@ const selectedTrack: Ref<TrackModel | null> = ref(null);
       class="absolute -inset-x-4 -inset-y-0 rounded-xl bg-tint"
     ></div>
 
+    <!-- eslint-disable vue/no-v-html -->
```

Comment:
Is disabling the eslint-rule here needed ..?

Reply:
wasnt able to turn it off in eslintrc 🤷‍♂️ tried for a while actually